src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2756,9 +2799,16 @@ namespace UnitarySystems {
                 state.dataSize->DataFractionUsedForSizing = 1.0;
                 SizingMethod = DataHVACGlobals::AutoCalculateSizing;
                 TempSize = DataSizing::AutoSize;
+                // pass design size to supplemental heater
+                state.dataSize->SuppHeatCap = max(this->m_DesignCoolingCapacity, this->m_DesignHeatingCapacity);
+            } else if (this->m_Humidistat && this->m_DehumidControlType_Num == DehumCtrlType::CoolReheat) {
+                state.dataSize->SuppHeatCap = max(this->m_DesignCoolingCapacity, this->m_DesignHeatingCapacity);
+            } else {
+                state.dataSize->SuppHeatCap = this->m_DesignHeatingCapacity;
             }
```

Comment:
The one issue with this is that this->m_DesignHeatingCapacity is the capacity of the heating coil. For a HP in a cooling dominated climate, this is the same size as the cooling coil (heating capacity was upsized to match). So this is not the raw heating load when a HP is used. If that HP does not use CoolReheat, do we want the supplemental heating coil capacity to match the heating coil, or be sized to meet the heating load? I just ran a local test suite to see what diffs this correction would cause (i.e., using saveRawHeatingCapacity here in the else). Only 1 additional example file had a diff, the HVACTemplate-5ZoneUnitarySystem file.

![image](https://user-images.githubusercontent.com/5778156/176205485-e57e4628-9e5a-418b-854c-ab44496d57c2.png)

Reply:
Ah, the UnitarySystem finds cooling = 5kW and heating = 10kW. The WSHP has it's own sizing routine and finds cooling = 5kW and sets heating cap = cooling cap. The UnitarySystem calls over to the WSHP and gets the coil sizes. So the 10 kW was the original heating capacity. Just another never ending thing that needs fixing. I'll post an issue.

Reply:
I'm surprised we can size anything in E+. Here's a single-zone HP of 5 kW size where total cooling = 5 kW and heating (supposedly) = 10 kW. The 5 kW HP has no problem meeting the winter design day zone load and holds Tstat temp. On the design days, the coils never exceed 3.5 kW although the cooling coil design sensible capacity is 3.8 kW, so that's close, and how these coils were sized. Is heating performance just luck? The heating size = 10 kW comes from a capacity calculation using the cooling design flow (e.g., main system flow = max(coolingFlow, heatingFlow) and a system Central Heating Design Supply Air Temperature = 50 C (Sizing:Zone SAT also = 50 C). The cooling air flow is 5 times larger than heating. At 5x flow, you don't need 50 C SAT to meet the zone heating load. That could be just dumb luck that the cooling size worked here. How often is the peak cooling load similar to the heating load? It seems HP sizing based on geography is a little more complex than first thought.

![image](https://user-images.githubusercontent.com/5778156/176257673-6281bceb-bf80-4b9e-aca1-ebf689258641.png)

System heating flows for Sys 4 from 2 design days shown in eplusssz.csv and zone air flow from epluszsz.csv:

![image](https://user-images.githubusercontent.com/5778156/176257503-b1602768-59eb-4f62-958f-53bdadf1f1f7.png)

When I request the system heating flow rate in UnitarySystem this is what I get. Someone's messin with me.

![image](https://user-images.githubusercontent.com/5778156/176259412-7a089a43-eaa4-4d51-ad7d-f2f541d6fef2.png)